packages/components/src/Combobox/components/ComboboxActivator/ComboboxActivator.tsx

```diff
@@ -3,22 +3,23 @@ import { Button } from "@jobber/components/Button";
 import { Chip } from "@jobber/components/Chip";
 import { ComboboxContext } from "../../ComboboxProvider";
 import { ComboboxActivatorProps } from "../../Combobox.types";
+import { useComboboxActivatorAccessibility } from "../../hooks/useComboboxActivatorAccessibility";
 
 export function ComboboxActivator(props: ComboboxActivatorProps) {
-  const { handleClose, open, setOpen } = React.useContext(ComboboxContext);
+  const { toggleOpen } = React.useContext(ComboboxContext);
+  const accessibilityAttributes = useComboboxActivatorAccessibility();
 
-  if (props.children.type === Button || props.children.type === Chip) {
+  if (
+    typeof props.children !== "function" &&
+    (props.children.type === Button || props.children.type === Chip)
+  ) {
     return React.cloneElement(props.children, {
-      role: "combobox",
```

Comment:
Just a comment for myself and anyone who looks at this in the future.
This exact logic has been moved to `useMakeComboboxHandlers`
https://github.com/GetJobber/atlantis/pull/2075/files#diff-09225568c5cd891a5fb7a1d356821d17dee48e6ffa609dbda4de510ad6478162R33-R44